app/assets/javascripts/govuk_publishing_components/analytics/scroll-tracker.js

```diff
@@ -5,21 +5,17 @@
 
   window.GOVUK = window.GOVUK || {}
 
+  var percentages = [
```

Comment:
Can this be a `const` Andy, or we not using those yet?

Reply:
Afraid not. `const` isn't supported by IE10 and below, and only partly supported in IE11 https://caniuse.com/const